src/main/scala/devices/tilelink/CLINT.scala

```diff
@@ -112,13 +101,19 @@ class CLINT(params: CLINTParams, beatBytes: Int)(implicit p: Parameters) extends
       timeOffset       -> RegFieldGroup("mtime", Some("Timer Register"),
         RegField.bytes(time, Some(RegFieldDesc("mtime", "", reset=Some(0), volatile=true))))
     )
+
+    def getOMRegMap(): OMRegisterMap = omRegMap
   }
+
+  val clintLogicalTree: CLINTLogicalTreeNode = new CLINTLogicalTreeNode(device, module.getOMRegMap)
```

Comment:
This could just be:
`val clintLogicalTree: CLINTLogicalTreeNode = new CLINTLogicalTreeNode(device, module.omRegMap)`

You don't need the extra getter function, and pretty much never should in Scala.